### ISSUE TITLE
11boot (latest change) : build failure (timeout)

### ISSUE THREAD

**Issue by TheFausap** [PDP-10/its#980]
Hello,

I did a git pull some minutes ago, and now the build fails with a timeout because the 11boot program fails to start (in some way) [I'm using KLH10]

```
*11boot$^K!
.VAL 0; G0001+3>>PUSHJ P,ALOAD
IT IS NOW  7:40:06 PM EDT, FRIDAY, JUN 15,2018

The latest command timed out.
Makefile:47: recipe for target 'out/klh10/rp0.dsk' failed

```
regards,
Fausto

**Comment by larsbrinkhoff** [comment]
Yes, I noticed that too and has a fix in the works: #975

**Comment by larsbrinkhoff** [comment]
And thanks for testing!

There should be a complete build running in GitLab, but apparently it didn't catch this.
